### PR TITLE
Instructions for installing Pop!_OS

### PR DESCRIPTION
**Pull request by ryanleesipes**
Be amazed at my awesome instructions for install Pop!_OS

### CODE REVIEW

**Comment by kylerschin** [comment]
Can we change the look of this? https://raw.githubusercontent.com/system76/pop-docs/344c9737319d21bc0577828490d2676ff880a521/images/install-pop-os/try-or-install-pop.jpg

**Comment by ryanleesipes** [comment]
@kylerschin yeah, I need a screenshot somehow of it. I'm sure there's a way to do it, I just snapped that with my camera.

**Comment by kylerschin** [comment]
@ryanleesipes here it is Mr. Fancy! ;)
https://raw.githubusercontent.com/dumptrump/sturdy-carnival/master/Screenshot%20from%202017-07-12%2008-18-34.png
Download this please.

**Comment by kylerschin** [comment]
https://raw.githubusercontent.com/dumptrump/sturdy-carnival/master/selected.png

**Comment by kylerschin** [comment]
@ryanleesipes hi ryan! I made a pull request to your repo. https://github.com/ryanleesipes/pop-docs/pull/3

**Comment by kylerschin** [comment]
https://github.com/ryanleesipes/pop-docs/pull/3

**Comment by ryanleesipes** [comment]
@cassidyjames can you review this?

File: docs/install-pop-os.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: Install Pop!_OS
+description: How to install the Pop!_OS
+---
+
+The following guide describes how to download the Pop\!\_OS .iso image, write it to a flash drive, and install it on the hardware of your choice.
```
Comment (cassidyjames) [inline docs/install-pop-os.md]: No need to escape the `!` fwiw.

File: docs/install-pop-os.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+Requirements: At the time of this writing Pop\!\_OS only runs on the x86 architecture. At least 4GB of RAM and 20GB of storage is recommended.
```
Comment (cassidyjames) [inline docs/install-pop-os.md]: Specifically, 64-bit, right?

File: docs/install-pop-os.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+
+![Flash Complete](/images/install-pop-os/flash-complete.png)
+
+Once the flash is complete (should look like the screenshot above), it's time to boot it up on the machine that you want to install Pop\!\_OS on!
+
+## Installing Pop\!\_OS
+
+We're finally to the part where we get to install Pop\!\_OS on your computer!
+
+Depending on the computer you are using, there are different buttons you have to press upon boot to get to choose the boot device. On a System76 computer, it's the F7 button. You want to just start pressing that as soon as the computer begins booting up, typically folks just keep pushing the button repeatedly until they see the boot menu come up. Once you see the boot menu, select your flash drive.
```
Comment (cassidyjames) [inline docs/install-pop-os.md]: I don't think that's true for every System76 device; should probably direct them to http://support.system76.com/articles/boot-menu/

File: docs/install-pop-os.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+Once the flash drives boots, you will be at the Pop\!\_OS desktop! You can use the operating system as you normally would now. Because it is running from the flash drive performance may be slower and if you reboot back into the flash drive at this point, you'll lose anything you've changed while running in this "live session".
+
+![Install Pop\!\_OS Icon](/images/install-pop-os/pop-live-desktop.jpg)
+
+First we have to select the language we want to use:
+
+![Select Language](/images/install-pop-os/select-language-pop.jpg)
+
+The next screen asks if you want to install updates during the OS install, if you are connected to a reliable Internet connection while installing, I recommend hitting that checkmark so you can have the new and shiny as soon as Pop\!\_OS is finished installing.
```
Comment (cassidyjames) [inline docs/install-pop-os.md]: For some reason I thought we removed this (or were planning to?). I could be wrong.

**Pull request merged**